app/assets/javascripts/timelines_select_boxes.js

```diff
@@ -29,6 +29,10 @@
 //requires 'autocompleter'
 
 jQuery(document).ready(function($) {
+  var formatSelection = function(item) {
```

Comment:
RFC: I saw similar code without the `var` keyword.
If I understand it correctly, the `var` keyword is what I want, because I want to limit the scope of this method. It shall not be available elsewhere.

Still not sure whether I really need that keyword in this specific case or not.